### ISSUE TITLE
Belt speeds are not accurate

### ISSUE THREAD

**Issue by masoudd** [tobspr/shapez.io#255]
My belt speed is 12/s. I filled it with stars and waited a couple of minutes. In the statistics tab it shows delivered stars as 588/m which is a lot less than 12*60=720/m. I'm not sure if this is actually a bug. Sorry if I'm missing something obvious.

**Comment by tobspr** [comment]
Hey, this is a problem once your base gets too big and the game can't keep up - thus its getting slower than the maximum speed. Does it persist if increasing the tick rate?

**Comment by masoudd** [comment]
> Does it persist if increasing the tick rate?

I'm sorry, I don't know how to change the tick rate?

**Comment by tobspr** [comment]
Settings > Target refresh rate

**Comment by masoudd** [comment]
Yes. I put it on 500Hz and it went up to 720.
I tested all tick rates:
500Hz -> 720
250Hz -> 684
165Hz -> 714
144Hz -> 612
100Hz -> 600
60Hz   -> 588

Curiously 250Hz makes a lower figure than 165Hz. I tested it twice to make sure.

I think this due to calculating the statistics from absolute time instead of per ticks and converting it to game seconds by the set ticks/seconds ratio.

I don't know if this is intended or not. It's just triggering my ocd honestly. Although it would be useful to find bottlenecks if for example you expect a full belt and it shows something less than the throughput of a full belt.
